src/app/utility/energy-consumption/energy-source/edit-meter-form/edit-meter-form.component.html

```diff
@@ -74,19 +83,30 @@ <h4>Edit Meter</h4>
                     <label for="heatCapacity">Heat Capacity <span class="required">*</span> </label>
                     <div class="input-group mb-3">
                         <input id="heatCapacity" class="form-control" type="number" formControlName="heatCapacity">
-                        <span class="unit"><span [innerHTML]="facilityEnergyUnit | settingsLabel"></span>/<span [innerHTML]="meterForm.controls.startingUnit.value | settingsLabel"></span>
+                        <span class="unit"><span [innerHTML]="facilityEnergyUnit | settingsLabel"></span>/<span
+                                [innerHTML]="meterForm.controls.startingUnit.value | settingsLabel"></span>
                         </span>
                     </div>
-                    <span class="inputError" *ngIf="meterForm.get('heatCapacity').hasError('required')">Heat capacity is required</span>
-                    <span class="inputError" *ngIf="meterForm.get('heatCapacity').hasError('min')">Value can't be less than 0</span>
+                    <span class="inputError" *ngIf="meterForm.get('heatCapacity').hasError('required')">Heat capacity is
+                        required</span>
+                    <span class="inputError" *ngIf="meterForm.get('heatCapacity').hasError('min')">Value can't be less
+                        than 0</span>
                 </div>
 
                 <div class="col">
                     <label for="siteToSource">Site to source <span class="required">*</span> </label>
                     <div class="mb-3">
                         <input id="siteToSource" class="form-control" type="number" formControlName="siteToSource">
                     </div>
-                    <span class="inputError" *ngIf="meterForm.get('siteToSource').hasError('required')">Site to source is required</span>
+                    <span class="inputError" *ngIf="meterForm.get('siteToSource').hasError('required')">Site to source
+                        is required</span>
+                </div>
+            </div>
+            <div class="row" *ngIf="meterFormDisabled">
+                <div class="col text-center">
+                    <span class="p-2 bg-warning">
```

Comment:
<p class="alert alert-warning"> Data found for this meter. Some meter properties cannot be changed. </p> instead of the "<span>".